app/models/hosting_environment.rb

```diff
@@ -4,15 +4,15 @@ module HostingEnvironment
   PRODUCTION_URL = I18n.t("service.url")
 
   def self.host
-    ENV.fetch("HOSTING_DOMAIN")
+    ENV.fetch("HOSTING_DOMAIN", "qualifications.lvh.me")
```

Comment:
I think we can drop the default values in this file — an environment which doesn't have these set should probably fail early. WDYT?